src/entities/entity.hpp

```diff
@@ -33,9 +33,6 @@ class Entity: public Displayable, public level::Collidable {
     // sarà definita a seconda del metodo d'attacco dell'entità
     virtual void attack(Entity *target);
 
-    // muove l'entità secondo la direzione impostata
-    void move(manager::Level *levelManager);
-
```

Comment:
🟥 i nemici si devono muovere, dovresti invece mantenere questa funzione, ma cercare di astrarre il modo con cui gestisci le collisioni. 

Quello che ti ho detto in chat :D